deploy/kubernetes/latest/node.yaml

```diff
@@ -2,46 +2,26 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: csi-node-sa
-  namespace: default
+  namespace: kube-system
 
 ---
 
 kind: ClusterRole
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: csi-node
-  namespace: default
```

Comment:
Why the node permission are all removed? If this is the case, Im wondering do we still need the service account of node daemon set?